class StoryError(Exception):
    pass


class FailureError(StoryError):
    pass
